app/(tabs)/page.tsx

```
@@ -4,15 +4,23 @@ import { useState } from 'react'
 import { User as UserIcon } from 'lucide-react'
+import { HomePageNew } from './page-new'
 
 export default function HomePage() {
+  const newUI = isNewUIEnabled()
+  
+  if (newUI) {
+    return <HomePageNew />
+  }
+
+  // Original UI
   const [selectedDate, setSelectedDate] = useState(new Date())
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)